R/dots.R

```diff
@@ -97,7 +97,8 @@ dots_clean_empty <- function(dots, is_empty, ignore_empty) {
 
   if (n_dots) {
     which <- match.arg(ignore_empty, c("trailing", "none", "all"))
-    switch(which,
+    switch(
+      which,
```

Comment:
Can we keep the `which` on the same line with the `switch(` here? This is one of the few special cases.